gazebo_plugins/worlds/gazebo_ros_wheel_slip_drawbar_pull_demo.world

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" ?>
+<!--
+  Gazebo ROS wheel slip plugin and drawbar pull demo
+
+  Try for example:
+
+  To publish time-varying drawbar pull force:
+
+    ros2 run gazebo_plugins wheel_slip_drawbar_pull_publisher.py
+
+  To change slip compliance of a group of wheels:
+
+    ros2 param set /trisphere_cycle_slip0/wheel_slip_rear slip_compliance_unitless_longitudinal 1.0
+    ros2 param set /trisphere_cycle_slip1/wheel_slip_rear slip_compliance_unitless_longitudinal 0.0
+
+  To get the current slip compliance of a group of wheels:
+
+    ros2 param get /trisphere_cycle_slip1/wheel_slip_rear slip_compliance_unitless_longitudinal
+-->
+<sdf version="1.6">
+  <world name="default">
+    <gravity>0 0 -9.8</gravity>
+
+    <include>
+      <uri>model://ground_plane</uri>
```

Comment:
If we are targeting : https://github.com/osrf/gazebo/pull/3164/files, we'll need ``<uri>model://plowing_effect_ground_plane</uri>``

Reply:
for purposes of a general demo, I think it's fine not to include plowing effects